robinhood.d.ts

```diff
@@ -0,0 +1,265 @@
+import request = require('request')
+
+declare function robinhood(options: robinhood.Options.WebApiOpts, callback: robinhood.InitCallback): robinhood.RobinhoodWebApi;
+
+declare namespace robinhood {
+
+  export namespace Options {
+    interface InitResponse {
+      mfa_required: boolean
+    }
+
+    interface WebApiOpts {
+      username?: string
+      password?: string
+      token?: string
+    }
+
+    interface OrdersOptions {
+      updated_at: string
+      instrument: string
+    }
+
+    interface EarningsOptionsWithSymbol {
+      range?: number
+      symbol: string
+    }
+
+    interface EarningsOptionsWithInstrument {
+      range?: number
+      instrument: string
+    }
+
+    interface BuySellOptions {
+      type: OrderType
+      quantity: number
+      bid_price: number
+      instrument: {
+        url?: string
+        symbol?: string
+      }
+      trigger: TriggerType
+      time: TimeInForceType
+    }
+  }
+
+  type TagType = TagTypes
+  type OrderType = OrderTypes
+  type IntervalType = IntervalTypes
+  type SpanType = SpanTypes
+  type TriggerType = TriggerTypes
+  type TimeInForceType = TimeInForceTypes
+  type InitCallback = (data?: Options.InitResponse) => void
+
+  enum TagTypes {
+    TopTen = '10-most-popular',
+    TopOneHundred = '100-most-popular'
+  }
+
+  enum TimeInForceTypes {
+    Immediate = 'immediate',
+    Day = 'day'
+  }
+
+  enum TriggerTypes {
```

Comment:
I am going to open an issue for this, but I think `TimeInForceTypes` and `TriggerTypes` are reversed according to https://github.com/sanko/Robinhood/blob/master/Order.md and the tests that I was doing the other day.  Side note - I wish 'ioc' time_in_force worked but maybe Robinhood deprecated it because I was getting invalid value.

Reply:
@chiefsmurph you're correct, not sure how I managed that! I can address this later PDT.

Reply:
Let me know when this is ready to merge and I'll release a new version with it